configs/xcache-redir/config.d/04-debug-tunning.cfg

```diff
@@ -0,0 +1,9 @@
+###################
+# This options are to be uncommented for debugging
+#
+##################
+
+#xrd.trace all
+#ofs.trace all
+#http.trace all
+#cms.trace all
```

Comment:
Are all of these really redirector specific? Maybe this should go in the main XCache package.

Reply:
No they are not I can move them there